tests/config/test_omegaconf_config.py

```diff
@@ -860,7 +860,7 @@ def test_not_hidden_config(self, tmp_path, hidden_path):
         assert not conf._is_hidden(hidden_path)
 
     def test_ignore_ipynb_checkpoints(self, tmp_path, mocker):
-        conf = OmegaConfigLoader(str(tmp_path), default_run_env=_BASE_ENV)
+        conf = OmegaConfigLoader(str(tmp_path), default_run_env="base")
```

Comment:
Why do you need to change it? isn't the default value `base` already?

Reply:
Accidentally left this in, reverted.